docs/use-events.md

```diff
@@ -1,13 +1,13 @@
 ---
-title: Find Events in the Catalog
+title: Event Catalog
 ---
 # About the Catalog
```

Comment:
Add `# Event Catalog` at the top and change "About the Catalog" to an h2: `## About the catalog`